src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -10,10 +11,11 @@ import { WithId } from "utils/id";
 import "./UserAvatar.scss";
 
 export interface UserAvatarProps {
-  user?: WithId<User>;
+  user?: WithId<User> | FirebaseReducer.Profile<User>;
```

Comment:
Why do we need to add `FirebaseReducer.Profile<User>`?

Reply:
It's the 'raw' type of `profile` that comes from `useUser`. We don't want to be adding it here like this. `WithId<User>` is the same thing, but better.